src/utils/api.ts

```diff
@@ -0,0 +1,43 @@
+import { RowCharacter, ModifiedCharacter, CharactersBySpecies } from '../types'
```

Comment:
This file can be under an independent `api` folder rather than util since you will have more logics that are tightly coupled with an API request/response treatment. `utils` get big and messy so try to avoid using this folder as much as you can by coupling some files under a single folder.

The following link is for an article that is about file structure for react project. Ngl I doesn't fully agree with it(especially its usage of `test` and `util`) but it generally makes sense. You might wanna casually check on API folder to see what it would look like.

https://dev.to/larswaechter/how-i-structure-my-react-projects-jii